### PR TITLE
Initial look of the quiz

### LINKED ISSUE #3: Starting page and initial upload of the question 1
- [x] Update the index.html
- [x] Update the style.css
- [x] Update the init.js
- [x] Write JS function getQuestion
- [x] Write JS listener for the start button
- [x] Write JS function showQuestion

### PR DESCRIPTION
**Pull request by aleks2407**
Added starting page, upload of the first question
Closes gh-3

### CODE REVIEW

File: src/logic/showQuestion.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+function showQuestion(nr) {
+    // Hide the setup
+    document.getElementById("explanation_div").style.display = "none";
+    // Show the game
+    document.getElementById("play_quiz_div").style.display = "block";
```
Comment (oguzkarademir) [inline src/logic/showQuestion.js:6]: in `init.js` these eventlistener codes already exist, I just wonder why you wrote it again ?
This branch works good, i am approving. Good work  💯 @aleks2407

**Review by oguzkarademir — APPROVED** [review]

File: src/logic/showQuestion.js (hunk 1 above)
Comment (aleks2407) [reply src/logic/showQuestion.js:6]: @oguzkarademir,in init.js we display the first DIV with the START button and hide the DIV with the question and rest of the buttons
Here we hide the DIV with START button and display the DIV with the question and rest of the buttons
Look more carefully on "block" and "none" , they are switched

Comment (oguzkarademir) [reply src/logic/showQuestion.js:6]: > @oguzkarademir,in init.js we display the first DIV with the START button and hide the DIV with the question and rest of the buttons
> Here we hide the DIV with START button and display the DIV with the question and rest of the buttons
> Look more carefully on "block" and "none" , they are switched

i understood,  thanks

**Review by Feruzteame — APPROVED** [review]
nice!

**Review by miroslavveljanoski — APPROVED** [review]
@aleks2407  this looks perfect to me. If you have time for an explanation, I have some questions about your logic 😄 
I'm going to merge this PR.

**Pull request merged**